scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -275,9 +280,23 @@ def _get_short_cid( container_id ):
     # https://docs.python.org/2/reference/expressions.html#slicings
     return container_id[:8]
 
+def _ignore_old_dead_container( container, threshold ):
+
+    result = False
+    # check for recently finished containers
+    if threshold is not None:
+        state = container.get( 'State', {} )
+
+        #ignore any that are finished and that are also too old
+        if state != 'running':
+            created = container.get( 'Created', 0 ) # default to a long time ago
+            if created < threshold:
+                result = True
+
+    return result
 
 def _get_containers(client, ignore_container=None, restrict_to_container=None, logger=None,
-                    only_running_containers=True, glob_list=None, include_log_path=False, include_k8s_info=False):
+                    only_running_containers=True, running_or_finished_since=None, glob_list=None, include_log_path=False, include_k8s_info=False):
```

Comment:
Add some argument documentation here as well.

Reply:
Done.  Also changed this to running_or_created_after to reflect that it's actually using the create time not the finish time to filter containers.